test/rules/structured-imports/ancestorCases-fix.ts.lint

```diff
@@ -0,0 +1,6 @@
+import * as SomethingAbsolute from 'something-absolute';
```

Comment:
I thought the `customRule-fix.ts.lint` files were for documentation. It being a corresponding fix results files makes sense but I'm not 100% sure that's the case.
What kind of gibberish did you write? The customRule just read and processes import statements.